csp/impl/wiring/numba_node.py

```diff
@@ -67,17 +67,17 @@ def _create(self, engine, memo):
         exposed_utility_values = {}
         cache_key = [self._impl]
         for (ts_idx, basket_idx), input in self.ts_inputs():
-            exposed_utility_values[
-                NumbaNodeParser.get_ts_input_value_getter_name(ts_idx)
-            ] = NumbaTSTypedFunctionResolver.get_value_getter_function(input.tstype.typ)
+            exposed_utility_values[NumbaNodeParser.get_ts_input_value_getter_name(ts_idx)] = (
```

Comment:
can you double check that you're using `ruff>0.3,<0.4`, i just did an autofix yesterday and i dont think this change should happen.

Reply:
ruff 0.3.4